packages/usa-link/src/usa-link.twig

```diff
@@ -2,7 +2,13 @@
 
 <p>This is <a class="usa-link usa-color-text-visited" href="javascript:void(0);">a visited link</a>.</p>
 
-<p>This is a link that goes to an <a class="usa-link usa-link--external" href="https://i.giphy.com/media/WPzQF6ruiIIVzHNlwX/source.gif">external website</a>.</p>
+<p>
+  This is a link that opens in <em>current</em> tab and goes to an <a class="usa-link usa-link--external" href="https://i.giphy.com/media/WPzQF6ruiIIVzHNlwX/source.gif">external website</a>.
```

Comment:
No action needed, just noting that opening `source.gif` is still a surprise every time

Reply:
Now that @amyleadem  mentions it yes, it does say "link" twice (which is tricky to discern since some of the links say "link" in them. 

Additionally, I hear JAWS trying to say the word "tab" which runs into the phrase "external website" both in ANDI and in JAWS. I'm attaching how it looks in ANDI so you can get an idea of how it sounds. Like a run-on sentence.

![external link text](https://user-images.githubusercontent.com/117778073/230920799-8fc1e0d3-85c7-4b8a-b900-47527f2fa558.png)

Reply:
@amycole501 @amyleadem I've added the following changes:

- Simplified the CSS
- Removed the base variable `$theme-external-link-sr-label`. 
- Added punctuation to the label ex: `"External, opens in a new tab."` to prevent screen readers from jumbling the words together.